control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -1118,9 +1123,17 @@ func secretHash(data []byte) string {
 }
 
 func (r *reconciler) reconcileOAuthServingCertCABundle(ctx context.Context, hcp *hyperv1.HostedControlPlane) error {
-	sourceBundle := cpomanifests.OpenShiftOAuthMasterCABundle(hcp.Namespace)
-	if err := r.cpClient.Get(ctx, client.ObjectKeyFromObject(sourceBundle), sourceBundle); err != nil {
-		return fmt.Errorf("cannot get oauth master ca bundle: %w", err)
+	var sourceBundle *corev1.ConfigMap
+	if util.HCPOAuthEnabled(hcp) {
+		sourceBundle = cpomanifests.OpenShiftOAuthMasterCABundle(hcp.Namespace)
+		if err := r.cpClient.Get(ctx, client.ObjectKeyFromObject(sourceBundle), sourceBundle); err != nil {
+			return fmt.Errorf("cannot get oauth master ca bundle: %w", err)
+		}
+	} else {
+		sourceBundle = cpomanifests.OIDCCAConfigMap(hcp.Namespace)
+		if err := r.cpClient.Get(ctx, client.ObjectKeyFromObject(sourceBundle), sourceBundle); err != nil {
```

Comment:
how's this used if no oauth enabled?

Reply:
this might be something I did to try to get components that need changes to work in the meantime